color/src/colorspace.rs

```diff
@@ -525,6 +525,60 @@ impl ColorSpace for Rec2020 {
     }
 }
 
+/// 🌌 The ACEScg color space.
+///
+/// This color space is defined by the Academy Color Encoding System [specification][acescg].
+///
+/// The ACEScg color space is a linear color space. The wide gamut makes this color space useful as
+/// a working space for computer graphics.
+///
+/// Other ACES color spaces include ACES2065-1, ACEScc, and ACEScct. This crate currently does not
+/// support these color spaces.
+///
+/// The ACEScg components are `[R, G, B]`. The components are bounded to `[-65504.0, 65504.0]`,
+/// though it is unusual to clip in this color space.
+///
+/// The conversion between D65 and the ACES whitepoint is done with the standard Bradford linear
+/// chromatic adaptation transform.
+///
+/// [acescg]: https://docs.acescentral.com/specifications/acescg/
+#[derive(Clone, Copy, Debug)]
+pub struct AcesCg;
+
+impl ColorSpace for AcesCg {
+    const IS_LINEAR: bool = true;
+
+    const TAG: Option<ColorSpaceTag> = Some(ColorSpaceTag::AcesCg);
+
+    fn to_linear_srgb(src: [f32; 3]) -> [f32; 3] {
+        // XYZ_to_lin_sRGB * ACESwp_to_D65 * ACEScg_to_XYZ
+        const ACESCG_TO_LINEAR_SRGB: [[f32; 3]; 3] = [
+            [1.705_051, -0.621_792_12, -0.083_258_87],
+            [-0.130_256_42, 1.140_804_8, -0.010_548_32],
+            [-0.024_003_36, -0.128_968_98, 1.152_972_3],
+        ];
```

Comment:
Yep, it was just a rounding error caused by rounding to eight decimal places in my calculations since I wasn't aware of Clippy's excessive precision lint. I re-ran the calculations and got the same result as you.